spec/spec_helper.rb

```diff
@@ -25,6 +24,10 @@
 
   config.example_status_persistence_file_path = "tmp/rspec_examples.txt"
   config.order = :random
+
+  config.include Capybara::DSL
+
+  FactoryBot.allow_class_lookup = false
```

Comment:
Why is this line needed?
And if it is needed, it should live in `spec/support/factory_bot.rb`